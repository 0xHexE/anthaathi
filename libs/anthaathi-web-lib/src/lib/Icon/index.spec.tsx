import * as React from 'react';
import { render } from '@testing-library/react';
import Anthaathi from './Anthaathi';

describe('Icons', () => {
  it('should render Anthaathi', () => {
    const icon = render(<Anthaathi />);
    expect(icon.container).toMatchSnapshot();
  });
});
