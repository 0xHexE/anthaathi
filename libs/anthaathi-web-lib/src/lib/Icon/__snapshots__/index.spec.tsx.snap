// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icons should render Anthaathi 1`] = `
<div>
  <svg
    viewBox="0 0 500 500"
    xmlns="http://www.w3.org/2000/svg"
  >
    <g
      data-name="Layer 2"
      id="Layer_2"
    >
      <g
        data-name="Layer 1"
        id="Layer_1-2"
      >
        <polygon
          class=""
          points="250 174.64 299.5 216.1 250 257.55 250 310.26 344.16 231.4 344.16 200.8 250 121.93 250 174.64"
        />
        <polygon
          class=""
          points="250 242.45 200.5 283.9 250 325.36 250 378.07 155.84 299.2 155.84 268.6 250 189.74 250 242.45"
        />
      </g>
    </g>
  </svg>
</div>
`;
